R/expect-equality.R

```diff
@@ -126,7 +126,8 @@ expect_waldo_equal_ <- function(
   exp,
   info = NULL,
   ...,
-  trace_env = caller_env()
+  trace_env = caller_env(),
+  error_prefix = NULL
```

Comment:
The addition of `error_prefix` here and in `expect_setequal_()` are done to avoid duplication of these function by allowing me to add `"Names of "`, otherwise I would need some gnarly `$lab` modifications